actionmailer/lib/action_mailer/queued_delivery.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActionMailer
+  module QueuedDelivery
+    extend ActiveSupport::Concern
+
+    included do
+      class_attribute :delivery_job, default: ::ActionMailer::MailDeliveryJob
```

Comment:
Why did we extract this module for two lines?

Reply:
It seemed clearer at the time to locate all queued-delivery concerns in a single place, but I agree that in retrospect it seems a bit much. Happy to move to `ActionMailer::Base` if that's preferred.

Reply:
Got it, I think I'm fine. It is not causing any trouble.